doc/source/whatsnew/v1.3.0.rst

```diff
@@ -849,6 +849,7 @@ Plotting
 - Prevent warnings when matplotlib's ``constrained_layout`` is enabled (:issue:`25261`)
 - Bug in :func:`DataFrame.plot` was showing the wrong colors in the legend if the function was called repeatedly and some calls used ``yerr`` while others didn't (partial fix of :issue:`39522`)
 - Bug in :func:`DataFrame.plot` was showing the wrong colors in the legend if the function was called repeatedly and some calls used ``secondary_y`` and others use ``legend=False`` (:issue:`40044`)
+- Bug in :meth:`BoxPlot._validate_color_args` in box plot when 'dark_background' theme was selected, caps or min/max markers for the plot was not visible (:issue:`40769`)
```

Comment:
pls change this to be a user friendly note, eg. DataFrame.plot.boxplot(....) or similar